.github/CONTRIBUTING.md

```diff
@@ -175,6 +175,22 @@ to verify ZFS is behaving as intended.
 
 ## Style Guides
 
+### Repository Structure
+
+OpenZFS uses a standardised branching structure.
+- The "development and main branch", is the branch all development should be based on.
+- "Release branches" contain the latest released code.
+- "Staging branches" contain selected commits prior to being released
+- Backports are special branches centered around supporting older releases by porting patches
```

Comment:
I'm not sure what this is referring to, I don't think we currently have any branches like this.  Though we certainly could at some point.

Reply:
Yes thats not clear at all, you are right.
I think it's not needed at all to reference backports seperately.

I could just change row 182 to:
"Release branches" contain the latest released code for said version of OpenZFS."

Which I think is clear enough.

Reply:
Done!